src/plugins/vis_type_timeseries/public/application/components/aggs/percentile_rank/percentile_rank.tsx

```diff
@@ -121,17 +126,36 @@ export const PercentileRankAgg = (props: PercentileRankAggProps) => {
             />
           </EuiFormRow>
         </EuiFlexItem>
-      </EuiFlexGroup>
-      <EuiSpacer />
-      {model.values && (
-        <PercentileRankValues
-          disableAdd={isTablePanel}
-          disableDelete={isTablePanel}
-          showOnlyLastRow={isTablePanel}
-          model={model.values}
-          onChange={handlePercentileRankValuesChange}
-        />
-      )}
+
+        <EuiFlexItem>
+          <EuiFormRow
+            label={
+              <FormattedMessage
+                id="visTypeTimeseries.percentileRank.values"
+                defaultMessage="Values"
+              />
+            }
+          >
+            <>
+              {model.values && (
```

Comment:
Why don't you add the whole EuiFormRow under model.values check here?

Reply:
Good question, I just moved it from the other place without thinking about it. I see that `model.values` are not optional fo this **agg** and should have a value. I think it was added cause in model it's an optional value....

Fixed